novu/enums/__init__.py

```diff
@@ -47,4 +48,5 @@
     "StepFilterType",
     "StepFilterValue",
     "TemplateVariableTypeEnum",
+    "StrEnum",
```

Comment:
Either this, if it stays, to keep the ordering
```suggestion
    "StrEnum",
    "TemplateVariableTypeEnum",
```

Or that if it gets removed
```suggestion
    "TemplateVariableTypeEnum",
```